### ISSUE TITLE
Task 2 looks weird

### ISSUE THREAD

**Issue by chilicoder** [new-hope/AngularJS_ITVDN#1]
When I open 
https://new-hope.github.io/AngularJS_ITVDN/Task_2.html

I see this
<img width="531" alt="screen shot 2017-02-23 at 10 06 59" src="https://cloud.githubusercontent.com/assets/3491833/23252122/ea234ac0-f9af-11e6-9ff0-52ad2a08013a.png">

Is it due to the thing that task is in progress?

**Comment by Grishchenko** [comment]
Я открыл, у меня все работает. Только один вопрос: дата как-то странно выглядит

**Comment by Grishchenko** [comment]
я попозже еще одну задачку выложу. Там простенький калькулятор надо было сделать.

**Comment by chilicoder** [comment]
видимо ты открываешь в Chrome. А я открываю в Safari и  там не работает. Поэтому когда делают интерфейс всегда спрашивают, в каких браузерах должно работать

и когда сообщают об ошибке, всегда спрашивают, в каком браузере открывали

1) попробуй открыть в Firefox. там тоже не работает
2) раз уж мы используем бутстрап, давай форму причешем. например как здесь
http://getbootstrap.com/css/#forms-horizontal

**Comment by Grishchenko** [comment]
Так норм?

**Comment by chilicoder** [comment]
да. так лучше. только ты в след раз через пул реквест изменения комить. так их легче отследить

**Comment by chilicoder** [comment]
а еще, если в pr написать closes #<номер issue> то ишью автоматом закроется
